projects/burnr/src/themes/substrate/dark.ts

```diff
@@ -0,0 +1,15 @@
+import { ThemeOptions } from '@material-ui/core/styles';
+
+const dark: ThemeOptions = {
+	palette: {
```

Comment:
I'd keep the primary & secondary theme colors from the old json files here (projects/burnr/src/themes/substrate/light.js) and also use the `colors.js` file.

That way you just need to change a color in the colors.js file once and it applies to both themes.

I took them from the brand deck. It's the pink and the cyan

https://github.com/paritytech/substrate-connect/pull/7/files#diff-2f0b32fbdf8fe51d80c3bcb4dd81d352L34
```
  primary: {
    light: colors.cyan.light,
    main: colors.cyan.main,
    dark: colors.cyan.dark,
    contrastText: colors.black
  },
  secondary: {
    light: colors.pink.light,
    main: colors.pink.main,
    dark: colors.pink.dark,
    contrastText: colors.white
  },
```

Reply:
will restore and get back!